packages/core/contracts/chainbridge/BeaconOracle.sol

```diff
@@ -1,144 +0,0 @@
-// SPDX-License-Identifier: AGPL-3.0-only
```

Comment:
Great call, this has been unused and here for way too long. Tbh, maybe we should have a quarterly purge of unused code. We can always get it back via the git history.